### PR TITLE
Add video feeds

### PR DESCRIPTION
**Pull request by KevSlashNull**
**DEPENDS ON https://github.com/strideynet/bsky-furry-feed/pull/249 TO BE MERGED**

---

This MR adds four video feeds:

- New Videos (Mixed SFW/NSFW)
- Hot Videos (Mixed SFW/NSFW)
- New Videos (NSFW only)
- Hot Videos (NSFW only)

For now, all videos are shown in the feeds without regards for hashtags.

### CODE REVIEW

File: feed/feed_test.go [hunk 1]
```diff
@@ -399,6 +423,29 @@ func TestPreScoredGenerator(t *testing.T) {
...
+				},
+			},
+			expectedPosts: []string{videoPost, nsfwVideoPost},
+		},
+		{
+			name: "tagged videos nsfw",
+			opts: preScoredGeneratorOpts{
+				Alg: "classic",
+				generatorOpts: generatorOpts{
+					Hashtags: []string{"art", "furryart"},
```
Comment (strideynet) [inline feed/feed_test.go:442]: Hmm should these hash tags be here ?

File: feed/feed_test.go [hunk 2]
```diff
@@ -399,6 +423,29 @@ func TestPreScoredGenerator(t *testing.T) {
...
+					IsNSFW:   tristate.True,
+					HasMedia: tristate.True,
```
Comment (strideynet) [inline feed/feed_test.go:444]: Should this be HasVideo instead of HasMedia ?

Comment (KevSlashNull) [reply feed/feed_test.go]: I think so yeah!

File: feed/feed_test.go (hunk 1 above)
Comment (KevSlashNull) [reply feed/feed_test.go:442]: Yeah, I wanted to test videos tagged with `#furryart`.

**Comment by KevSlashNull** [comment]
:rotating_light: Tests pass :rotating_light: 

I also added `HasMedia: tristate.False` to the video feeds, so they don’t include images. :sweat_smile:

**Review by strideynet — APPROVED** [review]

**Pull request merged**